packages/starter/example/components/SendTransaction.tsx

```diff
@@ -17,18 +17,18 @@ export const SendTransaction: FC = () => {
 
         let signature: TransactionSignature = '';
         try {
-            const transaction = new Transaction().add(
+            let recentBlockhash = await connection.getRecentBlockhash();
+            const transaction = new Transaction({ recentBlockhash: recentBlockhash.blockhash, feePayer: publicKey }).add(
```

Comment:
Hmm I'm not sure why this is necessary. We generally set this value if it's not provided in `sendTransaction`:

https://github.com/solana-labs/wallet-adapter/blob/a10828d74c3fae04e40722a3edc50e596f24b1b5/packages/core/base/src/signer.ts#L22

Can you say more about what's failing?

Reply:
I was just connecting a Phantom wallet and hitting the send button and it was failing, asking for the recent blockhash.

Reply:
Got it, thanks. Sounds like there's an underlying bug in the current implementation, I'll check it out.

Reply:
This has been fixed: https://github.com/solana-labs/wallet-adapter/blob/d26322bf91216583bcb983dab677f000c2c2d941/packages/wallets/phantom/src/adapter.ts#L190-L193